src/doom/g_game.c

```diff
@@ -381,6 +383,10 @@ void G_BuildTiccmd (ticcmd_t* cmd, int maketic)
 	    side += sidemove[speed]; 
 	if (joyxmove < 0) 
 	    side -= sidemove[speed]; 
+	if (mousebuttons[mousebturnright])
```

Comment:
Hm, or should I rather change these to `if (gamekeydown[key_right] || mousebuttons[mousebturnright])`? Because, as it is now both keys/buttons add to the `side` and `angleturn` variables, respectively, which are of course capped to reasonable min/max values later, but anyway...

Reply:
Would not doing that mean that players can do moves that original client doesn't allow like turning too fast? Probably safer to do it that way.